exercises/affine-cipher/description.md

```diff
@@ -0,0 +1,54 @@
+Create an implementation of the affine cipher,
+an ancient encryption system created in the Middle East.
+ 
+The affine cipher is a type of monoalphabetic substitution cipher.
+Each character is mapped to its numeric equivalent, encrypted with
+a mathematical function and then converted to the letter relating to
+its new numeric value. Although all monoalphabetic ciphers are weak,
+the affine cypher is much stronger than the atbash cipher,
+because it has many more keys.
+ 
+the encryption function is:
+ 
+  `E(x) = (ax + b) mod m`
+  -  where `x` is the letter's index from 0 - length of alphabet - 1
+  -  `m` is the length of the alphabet. For the roman alphabet `m == 26`.
+  -  and `a` and `b` make the key
+ 
+the decryption function is:
+ 
+  `D(y) = a^-1(y - b) mod m`
+  -  where `y` is the numeric value of an encrypted letter, ie. y = E(x)
+  -  it is important to note that a^-1 is the modal multiplicative inverse
+  -  the modal multiplicative inverse of a only exists if `a` and `m` are
+  -  coprime.
+ 
+To find the MMI of `a`:
+
+  `1 = an mod m`
+  -  where `n` is the modal multiplicative inverse of `a` mod `m`
```

Comment:
I have been trying to Google `modal multiplicative inverse` and only finding multiplicative inverse, which is simply a reciprocal.  I think it might be worth while to further simplify what is being asked for by trying to use simpler language, perhaps including a link to the definition of MMI that you used?  For example, Googling `MMI` was not helpful for me.

Reply:
I am confused by the language, to me it reads like you are using the word you are trying to define in the definition of the word you are trying to define....

Reply:
Okay. I will add sources for it.
In short it is literally the modular reciprocal. That's why it is also called `multiplicative inverse of x mod b` or whatever vars you want.  When I get home I have the link in my browser bookmarks.